index.html

```diff
@@ -1,9 +1,20 @@
 <!doctype html>
 <html class="no-js" lang="">
 <head>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
```

Comment:
Cool. I'll double check the JS thing. I 100% behind removing what is not needed. There might have been some other issue with my bad-Bootstrap code.